README.md

```diff
@@ -11,23 +11,28 @@ A live version of the API can be found [here](http://robokop.renci.org:5781/docs
 
 ## Local Development
 
-Docker-compose can be used to quickly set up a locally running version of the app. Once you have installed docker-compose you can run:
+### Management Script
+
+The codebase comes with a zero-dependency python management script that can be used to automate basic local development tasks. Make sure you have docker and docker-compose installed and then run:
 
 ```bash
-docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build
+python3 manage.py dev # starts server accessible at 5781
```

Comment:
Python 2 is officially dead, so I think it's safe to just use `python`. Alternatively, we could make `manage.py` executable with `#!/usr/bin/env python`.

Reply:
Python 2.7 is still installed on Mac OS by default (it's the default on my system) so leaving this as python3 would be convenient for me. I do think making the script executable would be good, I'll add that.